### PR TITLE
ENT - RMP-348 "Upon clicking on the search icon after entering a non-existing candidate in the search box, able to view candidates"

### PR DESCRIPTION
**Pull request by gayan-orangehrm**

### CODE REVIEW

File: components/src/core/components/CardTable/Table/TableHeaderSortDropdown.vue [hunk 1]
```diff
@@ -92,6 +92,7 @@ export default defineComponent({
   },
 
   methods: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (lgtkaushalya) [inline components/src/core/components/CardTable/Table/TableHeaderSortDropdown.vue:95]: We better avoid this

**Comment by lgtkaushalya** [comment]
Can we add some unit tests for this new scenario?

File: components/src/core/components/CardTable/Table/TableHeaderSortDropdown.vue (hunk 1 above)
Comment (gayan-orangehrm) [reply components/src/core/components/CardTable/Table/TableHeaderSortDropdown.vue]: Fixed

**Comment by gayan-orangehrm** [comment]
> Can we add some unit tests for this new scenario?

Wrote unit tests

**Pull request merged**